packages/database/src/migrations/20210719025659-AddFluTrackerFirstNationWithIliMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const REPORT_CODE = 'AU_FLUTRACKING_Percentage_First_Nations_ILI';
+
+const REPORT = {
+  id: generateId(),
+  code: REPORT_CODE,
+  config: {
+    fetch: {
+      aggregations: [
+        'FINAL_EACH_DAY',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'sub_district',
+            aggregationEntityType: 'district',
+          },
+        },
+        'MOST_RECENT',
+      ],
+      dataElements: ['FWV_LGA_004a', 'FWV_LGA_003a'],
+    },
+    transform: [
+      'keyValueByDataElementName',
+      {
+        transform: 'aggregate',
+        organisationUnit: 'group',
+        '...': 'last',
+      },
+      {
+        transform: 'select',
+        "'organisationUnitCode'": '$row.organisationUnit',
+        "'value'": 'divide($row.FWV_LGA_004a, $row.FWV_LGA_003a)',
```

Comment:
I suppose I'm slightly partial to us having these `value` fields still be human readable, even if it does get overwritten by web-config-server anyway. eg `"'% First nation participants with influenza like illness (ILI)'": 'divide($row.FWV_LGA_004a, $row.FWV_LGA_003a)'` but I'm honestly not fussed 👍